CMakeLists.txt

```diff
@@ -333,3 +343,14 @@ install(DIRECTORY ${ONNX_ROOT}/onnx DESTINATION include
 install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/onnx DESTINATION include
     FILES_MATCHING
     PATTERN "*.h")
+if(ONNX_ENABLE_GTESTS)
+    include_directories(
```

Comment:
I don't think this is needed (and also it's too broad, e.g. it's adding gtest include dir even when compiling non-test cc files). In unittest.cmake it already adds the include dirs by `target_include_directories(${_UT_TARGET} PUBLIC ${googletest_INCLUDE_DIRS} ${onnx_INCLUDE_DIRS} ${PROTOBUF_INCLUDE_DIRS})`